params:
  truck_info: 0  # LV:0 FV1:1 FV2:2
  udp_group_addr: 239.255.255.250
  udp_group_port: 9307 
  target_vel: 0.65
  safety_vel: 0.45
  lv_stop_dist: 0.5
  fv_stop_dist: 0.3
  safety_dist: 1.5
  target_dist: 0.8
  angle_degree: 0.0
  Kp_d: 2.0
  Ki_d: 0.2
